app/src/main/kotlin/com/flxrs/dankchat/main/MainFragment.kt

```diff
@@ -17,9 +17,8 @@ import android.widget.ProgressBar
 import android.widget.TextView
 import androidx.activity.OnBackPressedCallback
 import androidx.activity.result.PickVisualMediaRequest
-import androidx.activity.result.contract.ActivityResultContracts
-import androidx.activity.result.contract.ActivityResultContracts.*
-import androidx.activity.result.launch
+import androidx.activity.result.contract.ActivityResultContracts.PickVisualMedia
+import androidx.activity.result.contract.ActivityResultContracts.StartActivityForResult
```

Comment:
Unrelated changes

Reply:
seems Organizing Imports in Android Studio cause this change, FeelsDankMan

Reply:
should I revert tho?

Reply:
Weird, oh well